my-teachers/snippet.html

```diff
@@ -0,0 +1,55 @@
+<script>
+
+var user,$teachersList,myArray;
+
+user = FrogOS.getUser();
+
+$teachersList = $("<div>", {
+    id: Frog.Utilities.generateSimpleId()
+});
+$teachersList.html(
+    '<b>Your teachers:</b>'
+);
+
+myArray=[];
+
+
+/// Get all the groups assoicated with logged in user
```

Comment:
The grammar for single line notation in JavaScript is a double slash, not triple.

`// My comment`